pytests/tests/test_tdnf_python.py

```diff
@@ -93,64 +94,64 @@ def test_tdnf_python_install_single(utils):
     pkgs = [sglpkgname]
 
     tdnf_py_erase(pkgs=pkgs)
-    assert (utils.check_package(sglpkgname) == False)
+    assert (utils.check_package(sglpkgname) is False)
```

Comment:
use `not utils.check_package(sglpkgname)`, same for all the others. There is no need for comparisons.

Reply:
I think it's okay to use this and gradually change this when there is time and patience. This is being done in lot of places.

Reply:
Could probably done with a simple `sed` script. But does not have to be this change.

Reply:
I have replaced them all.